addons/binding/org.openhab.binding.xiaomivacuum/ESH-INF/thing/vacuumThing.xml

```diff
@@ -0,0 +1,225 @@
+				<description>Token for communication (in Hex)</description>
+				<default>FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF</default>
+			</parameter>
+			<parameter name="deviceId" type="text" required="false">
+				<label>Device ID</label>
+				<description>Device ID number for communication (in Hex)</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="1" max="9999" required="false">
+				<label>Refresh interval</label>
+				<description>Refresh interval for refreshing the data in seconds</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-group-type id="actions">
+		<label>Action</label>
+		<channels>
+			<channel id="vacuum" typeId="vacuum" />
+			<channel id="spot_clean" typeId="spot_clean" />
+			<channel id="return" typeId="return" />
+			<channel id="pause" typeId="pause" />
+			<channel id="commands" typeId="commands" />
+
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Status</label>
+		<channels>
+			<channel id="battery" typeId="system.battery-level" />
+			<channel id="clean_area" typeId="clean_area" />
+			<channel id="clean_time" typeId="clean_time" />
+			<channel id="error_code" typeId="error_code" />
+			<channel id="fan_power" typeId="fan_power" />
+			<channel id="in_cleaning" typeId="in_cleaning" />
+			<channel id="dnd_enabled" typeId="dnd_enabled" />
+			<channel id="map_present" typeId="map_present" />
+			<channel id="msg_seq" typeId="msg_seq" />
+			<channel id="msg_ver" typeId="msg_ver" />
+			<channel id="state" typeId="state" />
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="consumables">
+		<label>Consumables</label>
+		<channels>
+			<channel id="main_brush_time" typeId="main_brush_time" />
+			<channel id="side_brush_time" typeId="side_brush_time" />
+			<channel id="filter_time" typeId="filter_time" />
+			<channel id="sensor_dirt_time" typeId="sensor_dirt_time" />
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="dnd">
+		<label>Do not Disturb</label>
+		<channels>
+			<channel id="dnd_function" typeId="dnd_function" />
+			<channel id="dnd_start" typeId="dnd_start" />
+			<channel id="dnd_end" typeId="dnd_end" />
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="history">
+		<label>History</label>
+		<channels>
+			<channel id="total_clean_area" typeId="total_clean_area" />
+			<channel id="total_clean_time" typeId="total_clean_time" />
+			<channel id="total_clean_count" typeId="total_clean_count" />
+		</channels>
+	</channel-group-type>
+
+	<!-- Status channels -->
+	<channel-type id="clean_area">
+		<item-type>Number</item-type>
+		<label>Cleaning Area</label>
+		<state pattern="%.2d M2" readOnly="true" />
+	</channel-type>
+	<channel-type id="clean_time">
+		<item-type>Number</item-type>
+		<label>Cleaning Time</label>
+		<state pattern="%.2d min" readOnly="true" />
```

Comment:
%.2f min

The same applies to all `%.<number>d` below -> they should be `%.<number>f` because otherwise they conversion will fail and there will only be "Err" displayed in the UIs.